src/Writers/LaTeXWriter.jl

```diff
@@ -53,12 +60,13 @@ const DOCUMENT_STRUCTURE = (
     "subparagraph",
 )
 
-function render(doc::Documents.Document)
+function render(doc::Documents.Document, fmt::LaTeX)
```

Comment:
Currently, this needs to accept `Symbol` too, since `format` does not get replaced. It might be best to immediately in `Document()` to convert `:html` and `:latex` into the corresponding types. Together with a deprecation warning?